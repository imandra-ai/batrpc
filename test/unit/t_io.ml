open Batrpc_core

let str = "hello\n world\n\n123"

let () =
  let ic = new Io.In.bufferized @@ new Io.In.of_str str in
  let lines = Io.In.read_lines ic in
  Printf.printf "lines:\n";
  List.iter (Printf.printf "- %S\n") lines;
  flush stdout
