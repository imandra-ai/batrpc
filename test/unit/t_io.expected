lines:
- "hello"
- " world"
- ""
- "123"
